scripts/ray_autoscale.yaml

```diff
@@ -68,11 +68,20 @@ worker_nodes:
 
 setup_commands:
     - cd flow && git fetch && git checkout origin/master
-
-head_setup_commands:
+    - pip install ray==0.8.0
+    - pip install tabulate
     - pip install boto3==1.10.45  # 1.4.8 adds InstanceMarketOptions
     - pip install awscli==1.16.309
+    - pip install stable-baselines
     - pip install pytz
+    - pip install torch==1.3.1
+    - pip install tensorflow==2.0.0
+    - pip install lz4
+    - pip install dm-tree
```

Comment:
we don't want this here I think. has master actually been upgraded to 0.8.0?

Reply:
Ooh good catch I'll just take the ray install out